appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/model/EcosystemAppResponse.kt

```diff
@@ -8,7 +8,9 @@ import kotlinx.android.parcel.Parcelize
 @Parcelize
 data class EcosystemAppResponse(
         @SerializedName("version")
-        val version: String?,
+        val version: String="",
```

Comment:
format

Reply:
String comparison is more expensive than int

Reply:
cool